lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -756,6 +756,15 @@ def test_platform_language_is_used_for_logged_in_user(self):
             self.assertIn("You have been", subject)
             self.assertIn("You have been", message)
 
+    def test_platform_language_is_used_for_new_user(self):
+        # simulate a new user
+        subject, message = self.get_subject_and_message(None)
+        language_after_rendering = get_language()
+
+        self.assertIn("You have been invited in platform language", subject)
+        self.assertIn("You have been invited in platform language", message)
```

Comment:
This test fails because the string "You have been invited in platform language" doesn't appear at all in the subject or message. Please see https://github.com/edx/edx-platform/blob/master/docs/en_us/internal/testing.rst about how to run tests locally.